tests/203_dbaas_policy_add_connection_namespace.robot

```diff
@@ -0,0 +1,68 @@
+*** Settings ***
+Documentation       To verify DBaaSPolicy
+Metadata            Version    0.0.1
+
+Library             SeleniumLibrary
+Resource            ../resources/keywords/dbaas_policy.resource
+
+Suite Setup         Configure DBaaS Policy Namespaces
+
+Suite Teardown      Tear Down The Test Suite
+Test Teardown       Tear Down DBaaSPolicy Test Case
+
+Force Tags          UI    DBaaS
+
+
+*** Test Cases ***
+Scenario: DBaaSInventories On Source Namespace Is Not Available Without ConnectionNamespace And With Role Permission
+    [Tags]    dbaas-test-34    smoke
+    Given User Creates DBaaSPolicy As OCP_LDAP_PROJECT_ADM Without ConnectionNamespace And Imports MongoDB Provider Account
+    When User Creates ${connectionNs} And Aligns admin Role To The OCP_DBAAS_USER
+    And User Aligns admin Role To ${DBaaSTestNS} For OCP_DBAAS_USER
+    Then DBaaSInventory Is Not Available For ${connectionNs}
+
+Scenario: DBaaSInventories On Source Namespace Is Not Available To ConnectionNamespace And Without RBAC Permission
+    [Tags]    dbaas-test-35    smoke
+    Given User Creates DBaaSPolicy As OCP_LDAP_PROJECT_ADM With ConnectionNamespace And Imports MongoDB Provider Account
+    When User Creates ${connectionNs} And Aligns admin Role To The OCP_DBAAS_USER
+    Then The Namespace ${DBaaSTestNS} Is Not Available On The Project Dropdown
+
+Scenario: DBaaSInventories On Source Namespace Available To ConnectionNamespaces Accessed By Same User As Project Admin
+    [Tags]    dbaas-test-36    smoke
+    Given User Creates DBaaSPolicy As OCP_LDAP_PROJECT_ADM With ConnectionNamespace And Imports MongoDB Provider Account
+    When User Creates ${connectionNs} And Aligns admin Role To The OCP_LDAP_PROJECT_ADM
+    Then DBaaSInventory Is Available For ${connectionNS}
+
+Scenario: DBaaSInventories On Source Namespace Available To ConnectionNamespaces Accessed By Same User As Service Admin
+    [Tags]    dbaas-test-37    smoke
+    Given User Creates DBaaSPolicy As OCP_LDAP_PROJECT_ADM With ConnectionNamespace And Imports MongoDB Provider Account
```

Comment:
Shouldn't this user be OCP_LDAP_SERVICE_ADMIN like in the scenario title?

Reply:
The purpose of this scenario is to create a DBaaSPolicy to expose inventories to another namespace and accessing the another namespace with Service admin access with the same user. 
So on line 39, OCP_LDAP_PROJECT_ADM is given with "edit" role (service admin access) for the namespace mentioned on "ConnectionNamespace" field which fulfills the scenario.
Please ignore the user name as the RBAC configured for namespace.

Reply:
ok got it